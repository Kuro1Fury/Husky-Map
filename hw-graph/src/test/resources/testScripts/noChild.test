# Tests that a graph can be created, a node can be added, and listEdge works

CreateGraph A
AddNode A n1
ListChildren A n1