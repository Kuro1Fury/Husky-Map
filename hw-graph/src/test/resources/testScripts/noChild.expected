# Tests that a graph can be created, a node can be added, and listEdge works

created graph A
added node n1 to A
the children of n1 in A are: